packages/office-ui-fabric-react/src/components/SelectedItemsList/BaseSelectedItemsList.tsx

```diff
@@ -140,32 +145,27 @@ export class BaseSelectedItemsList<T, P extends IBaseSelectedItemsListProps<T>>
     return this.selection.getSelection() as T[];
   }
 
-  public UNSAFE_componentWillUpdate(newProps: P, newState: IBaseSelectedItemsListState): void {
-    if (newState.items && newState.items !== this.state.items) {
-      this.selection.setItems(newState.items);
+  public componentDidUpdate(oldProps: P, oldState: IBaseSelectedItemsListState) {
+    if (this.state.items && this.state.items !== oldState.items) {
+      this.selection.setItems(this.state.items);
     }
   }
 
   public componentDidMount(): void {
     this.selection.setItems(this.state.items);
   }
 
-  public UNSAFE_componentWillReceiveProps(newProps: P): void {
-    const newItems = newProps.selectedItems;
-
-    if (newItems) {
-      this.setState({ items: newItems });
-    }
-
-    if (newProps.selection) {
-      this.selection = newProps.selection;
-    }
-  }
-
   public render(): any {
     return this.renderItems();
   }
 
+  protected get selection(): Selection {
+    if (this.props.selection !== null && this.props.selection !== undefined) {
```

Comment:
You could do this with nullish coalescing (or just ||) too. `return this.props.selection || this._defaultSelection`